<?xml version="1.0"?>
<launch>

		<param name="use_sim_time" value="true" />

		<node name="my_loam" pkg="gpar_slam" type="my_loam3" output="screen" >
		<param name="map_resolution" value="0.05" />
		</node>

		<node name="rosbag" pkg="rosbag" type="play" args="$(find gpar_slam)/bags/LMS_nanook.bag --pause --clock" output="screen" />

		<node name="rviz" pkg="rviz" type="rviz" args="-d $(find gpar_slam)/rviz/myslam.rviz" />

		<node name="cloudtofile_map" type="cloudtofile" pkg="gpar_lidar" >
				<remap from="cloud" to="map_cloud" />
				<param name="reference" type="str" value="map" />
		</node>


		<node name="cloudtofile_scan" type="cloudtofile" pkg="gpar_lidar" >
				<remap from="cloud" to="cloud" />
				<param name="reference" type="str" value="cloud" />
		</node>	

</launch>

